docs/user/zvm/install_upi.md

```diff
@@ -253,14 +253,55 @@ INFO Waiting up to 30m0s for the bootstrap-complete event...
 The Cluster Image Registry [Operator][cluster-image-registry-operator] does not pick an storage backend for `None` platform. Therefore, the cluster operator will be stuck in progressing because it is waiting for the administrator to [configure][cluster-image-registry-operator-configuration] a storage backend for the image-registry.
 [NFS][openshift-nfs] should be picked as a [storage-backend][nfs-storage-backend].
 
-TODO - document NFS backend setup.
 
-Alternatively, for non-production clusters, emptyDir can be used for testing.
+#### Configuring NFS
+
+NFS is currently the only supported persistent storage option for OpenShift on Z.  To make an existing NFS share accessible for OpenShift to use as persistent storage, users must first attach it as a Persistent Volume.  At least 100GB of NFS storage space must be available for the image registry claim.
+
+```
+apiVersion: v1
+kind: PersistentVolume
+spec:
+  accessModes:
+  - ReadWriteMany
+  - ReadWriteOnce
+  capacity:
+    storage: 100Gi
+  nfs:
+    path: <NFS export path>
+    server: <ip of NFS server>
+  persistentVolumeReclaimPolicy: Recycle
+  volumeMode: Filesystem
+status: {}
+```
+
+Once the persistent volume is created, the image registry must be patched to use it.
+
+```
+oc patch configs.imageregistry.operator.openshift.io/cluster --type merge --patch '{"spec":{"storage":{"pvc":{"claim":""}}}}'
```

Comment:
Let's make sure we include the link to the documentation that provides the details for NFS for registry in the commit message/pr description.

Reply:
https://github.com/openshift/installer/blame/master/docs/user/zvm/install_upi.md#L253-L254 already points to upstream docs for configuring registry for NFS for baremetal in general.